src/DistributedMetadata/DDLService.cpp

```diff
@@ -415,10 +435,19 @@ void DDLService::mutateTable(IDistributedWriteAheadLog::RecordPtr record, const
     String user = block.getByName("user").column->getDataAt(0).toString();
     String payload = block.getByName("payload").column->getDataAt(0).toString();
 
-    std::vector<Poco::URI> target_hosts{toURIs(
-        placement.placed(database, table),
-        fmt::format(DDL_TABLE_PATCH_API_PATH_FMT, database, getURIEndpoint(record->headers), table),
-        http_port)};
+    std::vector<Poco::URI> target_hosts;
+    if (record->headers.contains("column"))
+    {
+        method = record->headers.at("query_method");
+        target_hosts = toURIs(placement.placed(database, table), getColumnApiPath(record->headers, database, table, method), http_port);
+    }
+    else
+    {
+        target_hosts = toURIs(
+            placement.placed(database, table),
+            fmt::format(DDL_TABLE_PATCH_API_PATH_FMT, database, getURIEndpoint(record->headers), table),
```

Comment:
rename `getURIEndpoint` to `getTableCategory()`

Reply:
fixed